xija/component/heat.py

```diff
@@ -384,6 +388,10 @@ def dvals_post_hook(self):
 
 class EarthHeat(PrecomputedHeatPower):
     """Earth heating of ACIS cold radiator (attitude, ephem dependent)"""
+
+    use_earth_vis_grid = True
```

Comment:
Should this be configurable?

Reply:
Good question.  My feeling was that we do not need to support this up through the level of a model-spec configurable variable.  I.e. I think we are pretty much just going to always run with the fast grid-based model going forward.  Even for model fitting you would want to use the grid model,  both for speed and for fidelity (fitting with the same code that you use for evaluation later).  Thoughts?

For debugging and validation it is already configurable in the sense that you can create a model and then set that class variable on-the-fly to change how the model then gets evaluated.

Reply:
Makes sense!